src/components/TextCase/index.js

```diff
@@ -0,0 +1,12 @@
+import { textCaseConstant } from "./constant";
+import styled from "styled-components";
+import { theme } from "../../styles";
+
+// props로 variant, color 를 받음
+
+export const TextCase = styled.div`
+  line-height: 100%;
+  font-style: normal;
+  ${({ variant }) => textCaseConstant[variant]};
+  color: ${({ color }) => theme.colors[color]}};
```

Comment:
```suggestion
  color: ${({ color,theme }) => theme.colors[color]}};
```
이렇게 작성하면 `theme`을 import 해오지 않아도 될 것 같은데 어떻게 생각하시나요?